tool/e2e_android.sh

```
@@ -890,6 +1354,38 @@ self_test() {
+  SERIAL_A='fixture-A'
+  LOG_BOUNDARY_A='watch-boundary'
+  LOADING_WATCH_PID=""
+  LOADING_WATCH_FILE=""
+  LOADING_WATCH_LEADER_PID=""
+  LOADING_WATCH_PROCESS_GROUP=""
+  LOADING_WATCH_SESSION=""
+  LOADING_WATCH_READY_FILE=""
+  local loading_watch_wait_index loading_watch_adb_pid
+  start_first_connect_loading_watch fixture-A \
+    || { printf 'loading watcher could not start in fixture\n' >&2; return 1; }
+  for loading_watch_wait_index in 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20; do
```

Shellcheck (0.11.0)

[warning] loading_watch_wait_index appears unused. Verify use (or export if used externally).

(SC2034)